Multi-Chain/kava-evm-cosmos-multi-chain/schema.graphql

```diff
@@ -0,0 +1,29 @@
+# To improve query performance, we strongly suggest adding indexes to any field that you plan to filter or sort by
+# Add the `@index` or `@index(unique: true)` annotation after any non-key field
+# https://academy.subquery.network/build/graphql.html#indexing-by-non-primary-key-field
+
+type Transfers @entity {
```

Comment:
Can you combined Transfers and Transactions into the same entity to better show the point of how we can combine data from two datasets?